charts/helm/templates/_helpers.tpl

```diff
@@ -2,6 +2,7 @@
 {{- omit .Values.git "url" "base" "branch" | toYaml}}
 url: $(.git.git.url | strings.ReplaceAll "ssh://git@" "https://")
 base: "$(.git.git.branch)"
+connection: {{.Values.git.connection}}
```

Comment:
I don't think this is required. It comes from `.Values.git`.

Reply:
It clashes with the connection key in the actions.gitops block

Reply:
it'll still clash if we add it in here

Reply:
its not since I remove it from the templates/playbook.yml file

Reply:
It's not coming from `templates/playbook.yml`. 

It's coming from the second line
```yaml
{{- omit .Values.git "url" "base" "branch" | toYaml}}
```

Reply:
Ahh, I see what you mean, fixed